src/mapkit.cs

```diff
@@ -327,6 +331,17 @@ interface MKMapItem : NSSecureCoding
 		[Export ("openMapsWithItems:launchOptions:"), Internal]
 		bool _OpenMaps ([NullAllowed] MKMapItem [] mapItems, [NullAllowed] NSDictionary launchOptions);
 
+		[iOS (13, 2), NoMac, NoTV, NoWatch]
+		[Introduced (PlatformName.UIKitForMac, 13, 2)]
+		[Export ("openInMapsWithLaunchOptions:fromScene:completionHandler:"), Internal]
+		void _OpenInMaps ([NullAllowed] NSDictionary  launchOptions, [NullAllowed] UIScene fromScene, Action<NSError> completionHandler);
+
+		[iOS (13, 2), NoMac, NoTV, NoWatch]
+		[Introduced (PlatformName.UIKitForMac, 13, 2)]
+		[Static]
+		[Export ("openMapsWithItems:launchOptions:fromScene:completionHandler:"), Internal]
+		void _OpenMaps ([NullAllowed] MKMapItem [] mapItems, [NullAllowed] NSDictionary launchOptions, [NullAllowed] UIScene fromScene, Action<NSError> completionHandler);
```

Comment:
Same as with the other method and either what @rolfbjarne said or me. Internal and the _ naming.